magento/pwa-studio
# PB Banner UI test

## Description

TODO: Describe your changes in detail here.

## Related Issue





Closes https://jira.corp.magento.com/browse/PWA-1540.

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Cypress `banner.spec.js` should pass
Steps - 
Go to venia-integration-test dir
Run `yarn install`
Run `yarn test`
Run `banner.spec.js` from cypress ui

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
